src/atoms/buttons/button.tsx

```diff
@@ -26,7 +26,7 @@ const Button = Styled.button<GenericStylingProps>`
   svg {
     width: 20px;
     height: 20px;
-    cursor: pointer;
+    cursor: pointer!important;
```

Comment:
We already enforce fill and clearly there was intention of forcing pointer, i don't think there will be case where icon of a button shouldn't have a pointer